src/App.tsx

```diff
@@ -1,15 +1,19 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.scss';
-import SignIn from './pages/signIn/SignIn';
+import React from "react";
+
+import logo from "./logo.svg";
+import "./App.scss";
+
+import SignIn from "./pages/signIn/SignIn";
+import SignUp from "./pages/signUp/SignUp";
 
 const App: React.FC = (): JSX.Element => {
   return (
-    <div className='App'>
-      <img src={logo} className='App-logo' alt='logo' />
+    <div className="App">
+      <img src={logo} className="App-logo" alt="logo" />
       <SignIn />
+      <SignUp />
```

Comment:
how you understand what component you must show ? or in this case you show component ```SignIn``` and ```SignUp``` ?

Reply:
in this case i show both components, iam going to separate it after implementing a routing